wgsl/index.bs

```diff
@@ -1039,15 +1039,44 @@ switch_stmt
   : SWITCH paren_rhs_stmt BRACKET_LEFT switch_body+ BRACKET_RIGHT
 
 switch_body
-  : CASE const_literal COLON BRACKET_LEFT case_body BRACKET_RIGHT
+  : CASE case_selectors COLON BRACKET_LEFT case_body BRACKET_RIGHT
   | DEFAULT COLON BRACKET_LEFT case_body BRACKET_RIGHT
 
+case_selectors
+  : const_literal (COMMA const_literal)*
+
 case_body
   :
   | statement case_body
   | FALLTHROUGH SEMICOLON
 </pre>
 
+A switch statement transfers control to one of a set of case clauses, or to a default clause,
+depending the evaluation of a selector expression of a scalar integer type.
+
+If the selector value equals a value in a case selector list, then control is transferred to
+the body of that case clause.
+If the selector value does not equal any of the case selector values, then control is
+transferred to the `default` clause.
+
+Each switch statement must have exactly one default clause.
+
+A literal value must not appear more than once in the case selectors for a switch statement.
+
+The case selector values must have the same type as the selector expression.
+
+Note: A literal value must not be duplicated within a case clause, and must not appear in more
+than one case selector list.
```

Comment:
Isn't this exactly what you said above? "A literal value must not appear more than once in the case selectors for a switch statement.". I think that covers duplication within a single case, as well as appearing in only one case.

Reply:
ok. will remove. (This was already merged as part of the parent.)